test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -299,10 +299,10 @@ Date: REGEX(.*)
     "requestSemaphoreWaitingTime": "REGEX(\d+.\d+)",
     "statisticsRequests": "2",
     "subCache": "REGEX([0-9a-f, ]{50})",
-    "subCacheInserts": "2",
+    "subCacheInserts": "3",
```

Comment:
Yes, there are LOTS of changes in the statistics. Now ALL subs are in the cache, and counted.  Before only the patterned were there ...